server/mailer/index.js

```diff
@@ -0,0 +1,35 @@
+const SendGridMail = require('@sendgrid/mail');
+
+const mailTemplate = require('./mail-template');
+
+const { SENDGRID_API_KEY } = process.env;
+
+SendGridMail.setApiKey(SENDGRID_API_KEY);
+
+const sendInvitation = (req, resp) => {
+  const { email: receiver } = req.body;
+  const {
+    user: { email: sender }
+  } = req;
+
+  const message = {
+    to: receiver,
+    from: sender,
+    subject: 'Join EOC!',
+    html: mailTemplate(receiver, sender)
```

Comment:
this should also take as param host. Can be taken from 'req', and remove TEMP_DEV